File: server/Models/Recipe.cs
namespace allspice.Models;

public class Recipe
{
    public int id { get; set; }

    public DateTime createdAt { get; set; }

    public DateTime updatedAt { get; set; }

    public string title { get; set; }

    public string instructions { get; set; }

    public string img { get; set; }

    public string category { get; set; }

    public string creatorId { get; set; }

    public Profile creator { get; set; }
}

File: client/src/models/Recipe.js
import { Account } from "./Account.js"

export class Recipe{
    constructor(data){
        this.id = data.id || data._id
        this.createdAt = new Date(data.createdAt)
        this.updatedAt = new Date(data.updatedAt)
        this.title = data.title
        this.instructions = data.instructions
        this.img = data.img
        this.category = data.category
        this.creatorId = data.creatorId
        this.creator = new Account(data.creator)
        this.subTitle = data.subTitle
    }
}